man/systemd.network.xml

```diff
@@ -776,6 +776,31 @@ Table=1234</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IPv4ReversePathFilter=</varname></term>
+        <listitem>
+          <para>Configures IPv4 Reverse Path Filtering. Reverse Path Filtering is used to prevent packets that arrived
+          via one interface from leaving via a different interface. When a machine with reverse path filtering enabled
+          receives a packet, the machine will first check whether the source of the received packet is reachable through
+          the interface it came in. If it is routable through the interface which it came, then the machine will accept the packet.
+          If it is not routable through the interface which it came, then the machine will drop that packet. Takes one of
```

Comment:
This description uses two terms: "reachable" and "routable". I think it'd be easier to understand if the text was more direct:
> Configure IPv4 Reverse Path Filtering. If enabled, when an IPv4 packet is received, the machine will first check whether the `<emphasis>source</emphasis>` of the packet would be routed through the interface it came in. If there is no route to the source on that interface, the machine will drop the packet.